draft-ietf-dots-architecture.md

```diff
@@ -1065,8 +1062,7 @@ perspective offers a model for alternative or unanticipated triggers for DOTS
 deployments. In all cases, what network conditions merit a mitigation request
 are at the discretion of the DOTS client operator.
 
-The interfaces required to trigger the mitigation request in the following
-scenarios are implementation-specific.
+The mitigation request itself is defined by DOTS, however the interfaces required to trigger the mitigation request in the following scenarios are implementation-specific. .
```

Comment:
Remove extra period at the end of the sentence.

Reply:
Ok